components/data_comps/datm/bld/build-namelist

```diff
@@ -381,9 +381,7 @@ if ( $GRID eq "CLM_USRDAT" && $CLM_USRDAT_NAME =~ /^(UNSET|)$/ ) {
 EOF
 }
 
-(-d $DIN_LOC_ROOT)  or  die <<"EOF";
-** $ProgName - CESM inputdata root is not a directory: \"$DIN_LOC_ROOT\" **
-EOF
+(-d $DIN_LOC_ROOT)  or mkdir $DIN_LOC_ROOT;
```

Comment:
I do not believe that this is an appropriate place to create the DIN_LOC_ROOT directory.   It should have already been created when this script was called and erroring out is the appropriate behavior here.